### ISSUE TITLE
Something about finetune

### ISSUE THREAD

**Issue by lzhptr** [LinWeizheDragon/FLMR#37]
Where can I see the current losses? Is it the train/loss_step field?
![image](https://github.com/user-attachments/assets/9ac4bb06-062d-443f-b600-cc85cefea186)

But it looks unstable，thank u!

**Comment by LinWeizheDragon** [comment]
train/loss_step is the loss of the current step. You can find the loss plot in the wandb log.
The loss is usually unstable. This is because the model is already pre-trained on large-scale data and if your new data is not too challenge, the starting loss is already small. It takes some time for the model to find the right loss descend. This results in a loss curve that shoots up first and then decreases to converge.

You can find a loss_epoch field which shows you the aggregated loss over an entire epoch, which is more stable.
![image](https://github.com/user-attachments/assets/3b045414-6e36-4831-85a1-b4206c65702c)

**Comment by lzhptr** [comment]
> train/loss_step is the loss of the current step. You can find the loss plot in the wandb log. The loss is usually unstable. This is because the model was already pre-trained on large-scale data and if your new data is not too challenging, the starting loss is already small. It takes some time for the model to find the right loss descent. This results in a loss curve that shoots up first and then decreases to converge.
> 
> You can find a loss_epoch field which shows you the aggregated loss over an entire epoch, which is more stable. ![image](https://private-user-images.githubusercontent.com/33350454/382787363-3b045414-6e36-4831-85a1-b4206c65702c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA3NzY3NjEsIm5iZiI6MTczMDc3NjQ2MSwicGF0aCI6Ii8zMzM1MDQ1NC8zODI3ODczNjMtM2IwNDU0MTQtNmUzNi00ODMxLTg1YTEtYjQyMDZjNjU3MDJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA1VDAzMTQyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFjNzFiMzE4Yzg0NDY5ZDRjYThiMmViNWQ3NzlkZGFhMDQwZGE4YTYwN2RkYTA5MmQ5MjU1ODI0YWEzZWZiOTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.G2zojTz8r09IM4gNnLKzc5sGHmGrtsBWG57MHutFv4Q)

thank u ! But why is my curve like this?
![image](https://github.com/user-attachments/assets/ade5c5ad-53d3-40e7-a5ac-5ff97626d093)
My valid/loss epoch curve is like this
![image](https://github.com/user-attachments/assets/837223bb-b3a6-4394-87b4-133254d2e687)

**Comment by LinWeizheDragon** [comment]
The epoch-level train loss is logged per epoch. This means you haven't finished the second epoch.
The validation loss looks correct. It is logged per validation and per end of epoch. You can perform indexing and retrieval to obtain the actual model performance.
